src/utilities.js

```diff
@@ -1,10 +1,76 @@
+
 const checkResponse = (response) => {
   if(response.ok) {
   return response.json()
 } else {
   throw new Error('this request isn\'t available')
 }}
 
+export const postWatchlist = (watchListObj) => {
+  return fetch('https://what-2-watch-be.herokuapp.com/v1/watchlists/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(watchListObj)
+  })
+}
+
+export const postThumb = async ({api_movie_id, up, title}) => {
```

Comment:
This is a beast. I don't think there's much we can do to refactor but we should keep an eye out.